ste/xfer-remoteToLocal.go

```diff
@@ -73,6 +73,17 @@ func remoteToLocal(jptm IJobPartTransferMgr, p pipeline.Pipeline, pacer pacer, d
 		}
 	}
 
+	if jptm.MD5ValidationOption() == common.EHashValidationOption.FailIfDifferentOrMissing() {
+		// We can make a check early on MD5 existence and fail the transfer if it's not present.
+		// This will save hours in the event a user has say, a several hundred gigabyte file.
+		if len(info.SrcHTTPHeaders.ContentMD5) == 0 {
+			jptm.LogDownloadError(info.Source, info.Destination, errExpectedMd5Missing.Error(), 0)
```

Comment:
Does `LogDownloadError` emit these errors to the `azcopy` stdout? The previous error for missing MD5 was only emitted to the log file instead of the more user accessible stdout.

Reply:
No, it logs them to the job log.

Reply:
Should errors like these reach the stdout? 

When running I was thinking it was a transient issue before looking further into the logs, and finding the source files didn't have an MD5 value. Emitting the actual error to the stdout would let me know the actual error more quickly.

Reply:
I suspect that question is a bigger, and more general one.  I'd guess that for this PR, we should stay consistent with existing conventions in the app. If one day, we hypothetically LogDownloadError to use stderr, then that would take care of not just this case, but all other relevant ones where stderr could be used in this way.